perses/app/cli.py

```diff
@@ -72,99 +84,30 @@ def _test_platform(platform_name):
-    yaml_parse_name = f"parsed-{time}-{yaml_name}"
-
-    # First make files in same dir as yaml
-    files_next_to_yaml = [
-        "debug.png",
-        "system.xml",
-        yaml_parse_name,
-    ]
-
-    for _file in files_next_to_yaml:
-        with open(_file, "w") as fp:
-            pass
-
-    # Now we make the directory structure
-    trajectory_directory = Path(options["trajectory_directory"])
-    dirs_to_make = trajectory_directory.joinpath("xml")
-    Path(dirs_to_make).mkdir(parents=True, exist_ok=True)
-
-    # Now files that belong in the lower directories
-    files_in_lower_dir = [
-        "atom_mapping.png",
-        "out-complex_checkpoint.nc",
-        "out-complex.nc",
-        "out-complex.pdb",
-        "outhybrid_factory.npy.npz",
-        "out-solvent_checkpoint.nc",
-        "out-solvent.nc",
-        "out-solvent.pdb",
-        "out_topology_proposals.pkl",
-        "out-vacuum_checkpoint.nc",
-        "out-vacuum.nc",
-    ]
-
-    # add the dir prefix
-    files_in_lower_dir = [
-        Path(trajectory_directory).joinpath(_) for _ in files_in_lower_dir
-    ]
-
-    for _file in files_in_lower_dir:
-        with open(_file, "w") as fp:
-            pass
-
-    # Now the files in the 'xml' dir
-    files_in_xml_dir = [
-        "complex-hybrid-system.gz",
-        "complex-new-system.gz",
-        "complex-old-system.gz",
-        "solvent-hybrid-system.gz",
-        "solvent-new-system.gz",
-        "solvent-old-system.gz",
-        "vacuum-hybrid-system.gz",
-        "vacuum-new-system.gz",
-        "vacuum-old-system.gz",
-    ]
-
-    # add the dir prefix
-    files_in_xml_dir = [dirs_to_make.joinpath(_) for _ in files_in_xml_dir]
-
-    for _file in files_in_lower_dir:
-        with open(_file, "w") as fp:
-            pass
-
-
 @click.command()
 @click.option("--yaml", type=click.Path(exists=True, dir_okay=False), required=True)
 @click.option("--platform-name", type=str, default=None)
-def cli(yaml, platform_name):
+@click.option("--override", multiple=True, required=False, default=None)
+def cli(yaml, platform_name, override):
     """test"""
     click.echo(click.style(percy, fg="bright_magenta"))
     click.echo("📖\t Fetching simulation options ")
-    options = getSetupOptions(yaml)
+    options = getSetupOptions(yaml, override_string=override)
     click.echo("🖨️\t Printing options")
     click.echo(options)
+    if override:
+        click.echo("✍️ \t Overrides used")
     click.echo("🕵️\t Checking OpenEye license")
     _check_openeye_license()
     click.echo("✅\t OpenEye license good")
     click.echo("🖥️⚡\t Checking whether requested compute platform is available")
     _test_platform(platform_name)
-    click.echo("🖨️\t Writing out files")
-    trajectory_directory = options["trajectory_directory"]
-    _write_out_files(trajectory_directory, options)
+    click.echo("🏃\t Running simulation")
+    run(yaml_filename=yaml, setup_options=options)
```

Comment:
Shouldn't it be simpler and enough if we just add an `override_string` parameter to the `run` function itself? Because the latter still calls `getSetupOptions` and then that one would accept the `override_string` parameter and deal with it. That would mean we only have to call `run` here, and also it would solve potential issues in `run` mentioned in the comments for `setup_relative_calculation.py`, by always requiring to call `getSetupOptions`.

Reply:
Maybe `getSetupOptions` shouldn't be a public API point at all?

Reply:
Really nothing in `setup_relative_calculation.py` should be public. I think I can re-work things so that we only have to call run. I used this approach since when this was just a demo, I never called `run` just `getSetupOptions`.

While I don't think we _need_ to do this, I think it will make things a bit more simple so I will make this modification.

Reply:
Okay so I made the changes, but now there isn't an easy way to print the options + overrides before the simulation runs, is this change still worth it?

Reply:
Yes, I see and I agree! I was thinking about this because the benchmark code does use the `run()` function that way. Hence why it seemed simpler to me to try using just one function from the `setup_relative*.py`, even if we agree we shouldn't use any of them that way.